lib/components/user/monitored-trip/trip-summary-pane.tsx

```diff
@@ -1,61 +1,190 @@
-import { FormattedList, FormattedMessage } from 'react-intl'
+import { Bell, BellSlash, Calendar, Clock } from '@styled-icons/fa-regular'
+import { FormattedMessage } from 'react-intl'
 import React from 'react'
 
 import { dayFieldsToArray } from '../../../util/monitored-trip'
 import { MonitoredTripProps } from '../types'
-import FormattedDayOfWeek from '../../util/formatted-day-of-week'
-import Strong from '../../util/strong-text'
+import LocationIcon from '@opentripplanner/location-icon'
+import styled from 'styled-components'
 
-import TripSummary from './trip-summary'
+import { InlineLoading } from '../../narrative/loading'
+
+import MonitoredDays from './trip-monitored-days'
+import TripSummary from './trip-duration-summary'
 
 /**
  * Displays the summary information of a monitored trip.
  */
+
+const SavedTripBody = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 0 15px;
```

Comment:
We might be able to remove this padding, as the inner content already has generous padding.

Reply:
Do you have a preference for having the location icons flush or indented with respect to the the trip name heading?

Reply:
I prefer it indented, especially with the trip details being wider than the location details, I think it creates the impression that they're even.